sdk/ml/azure-ai-ml/azure/ai/ml/entities/__init__.py

```diff
@@ -284,6 +284,7 @@
     "RecurrenceTrigger",
     "RecurrencePattern",
     "JobSchedule",
+    "Schedule",
```

Comment:
It doesn't look like you've updated the signature of the schedule operations yet - so would be good to get a sense of the scenarios around schedules to know whether this would be the right approach.
Essentially - what do people _do_ with schedules? How do the use these objects?
If I have production code that does something like this:
```python
# Enable all job schedules
for schedule in client.schedules.list():
    client.schedules.begin_enable(schedule.name)
```
Am I going to be broken if `list` starts returning new schedule type objects I didn't expect? I'm assuming that's unlikely to happen unless I'm actually choosing to _create_ those new schedule types in the first place?
What other use-cases do you support for schedules? I'm trying to think of cases where I'm blindly listing everything and passing them to another process or operation (e.g. listing from one workspace and passing them to `create` in a different workspace) - these are the scenarios that might be broken in the case of changing the output type. And while this wouldn't present as a runtime error like suddenly getting `AttributeError`, because the API has been kept consistent. But if any of the new Schedule types cannot be used in the same way as a `JobSchedule`, and I don't know if I might get them, then that potentially could be a problem.
If, on the other hand, I'm only going to start seeing new `Schedule` types returned from list if these are new features that I'm opting into - that would be less likely to break someone.... Just trying to make sure I have the full picture :)

Reply:
Resolution we agreed on:

`create_job` is a property of `Schedule` base class. By default it outputs a warning to the user saying it is not a supported property. `JobSchedule` overrides this property to return the actual `create_schedule` value since this field only applies to that schedule so far. Customers only see new schedules if the new schedules are created **only in their** workspace by them or someone else.